packages/react/src/auto/polaris/submit/PolarisAutoSubmit.tsx

```diff
@@ -0,0 +1,9 @@
+import { Button } from "@shopify/polaris";
+import React, { ReactNode } from "react";
+
+export const PolarisAutoSubmit = (props: { children?: ReactNode }) => {
+  // const { submit } = useAutoFormMetadata();
+  // The submit function from useAutoFormMetadata can be used instead of the `submit` property on Button
```

Comment:
Maybe move these two comments above and make it as a comment block? Like this:
```ts
/**
 * (the comments)
 */
export const PolarisAutoSubmit = (props: { children?: ReactNode }) => {
```

Reply:
Updated comment position